app/models/match_decisions/confirm_housing_subsidy_admin_decline_dnd_staff.rb

```diff
@@ -85,9 +85,9 @@ def decline_confirmed
         # TODO maybe rerun the matching engine for that vacancy and client
       end
 
-      def canceled
+      def canceled contact
         Notifications::MatchCanceled.create_for_match! match
-        match.canceled!
+        match.canceled! contact&.id
```

Comment:
`ClientOpportunityMatch#canceled!` takes a named parameter, so this won't work as written.